jupyterhub/proxy.py

```diff
@@ -1,12 +1,21 @@
-"""API for JupyterHub's proxy."""
+"""API for JupyterHub's proxy.
+
+Route Specification:
+
+- A routespec is a URI (excluding scheme), e.g.
+  'host.name/path/' for host-based routing or '/path/' for default routing.
```

Comment:
It's probably host.name:port. Most code that does splits should handle this just fine, but we should explicitly mention it here I think.